sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRepositoryAsync.java

```diff
@@ -249,25 +249,25 @@ public RegistryArtifactAsync getArtifact(String digest) {
      * @throws HttpResponseException thrown if any other unexpected exception is returned by the service.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedFlux<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrderBy orderBy) {
+    public PagedFlux<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrder orderBy) {
         return new PagedFlux<>(
             (pageSize) -> withContext(context -> listManifestPropertiesSinglePageAsync(pageSize, orderBy, context)),
             (token, pageSize) -> withContext(context -> listManifestPropertiesNextSinglePageAsync(token, context)));
     }
 
-    PagedFlux<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrderBy orderBy, Context context) {
+    PagedFlux<ArtifactManifestProperties> listManifestProperties(ArtifactManifestOrder orderBy, Context context) {
         return new PagedFlux<>(
             (pageSize) -> listManifestPropertiesSinglePageAsync(pageSize, orderBy, context),
             (token, pageSize) -> listManifestPropertiesNextSinglePageAsync(token, context));
     }
 
-    Mono<PagedResponse<ArtifactManifestProperties>> listManifestPropertiesSinglePageAsync(Integer pageSize, ArtifactManifestOrderBy orderBy, Context context) {
+    Mono<PagedResponse<ArtifactManifestProperties>> listManifestPropertiesSinglePageAsync(Integer pageSize, ArtifactManifestOrder orderBy, Context context) {
         try {
             if (pageSize != null && pageSize < 0) {
                 return monoError(logger, new IllegalArgumentException("'pageSize' cannot be negative."));
             }
 
-            final String orderByString = orderBy == ArtifactManifestOrderBy.NONE ? null : orderBy.toString();
+            final String orderByString = orderBy == ArtifactManifestOrder.NONE ? null : orderBy.toString();
```

Comment:
Would there be a time where `NONE` and null are treated differently?

Reply:
no